plugins/kafka/kafka.cpp

```diff
@@ -475,7 +475,14 @@ namespace KafkaPlugin
     {
         if (message.err() != RdKafka::ERR_NO_ERROR)
         {
-            DBGLOG("Kafka: Error publishing message: %d (%s); message: '%s'", message.err(), message.errstr().c_str(), static_cast<char*>(message.payload()));
```

Comment:
There are three things handled here:  1) null-terminating the message, 2) making sure a null pointer isn't passed to DBGLOG, and 3) providing an explicit '<no message>' value if there is a null pointer.  Theoretically, the DBGLOG here should be called only when there is a problem, so my thought was that an extra copy would not be too much of an imposition, and that this code is an easily-maintained way of accomplishing all the goals.  

Given all of that, would you recommend that I use %.*s and inline everything?